subsys/bluetooth/host/conn.c

```diff
@@ -2249,9 +2249,11 @@ int bt_conn_auth_pairing_confirm(struct bt_conn *conn)
 }
 #endif /* CONFIG_BT_SMP || CONFIG_BT_BREDR */
 
-u8_t bt_conn_get_id(struct bt_conn *conn)
+u8_t bt_conn_index(struct bt_conn *conn)
 {
-	return conn - conns;
+	u8_t index = conn - conns;
```

Comment:
I think you might get coding style warnings for this - there should be an empty line after the function variable declarations and the rest of the function body.

Reply:
I have added the extra line now